samples/vars/storage_fileset_vars.yml

```diff
@@ -0,0 +1,32 @@
+---
+#
+# saples/vars/storage_fileset_vars.yml
+#
+
+# Refer to `man mmchfs` and `man mmchnsd` man pages for a description of these
+# storage parameters. The `filesystem` parameter is mandatory, `servers`, and
+# the `device` parameter is mandatory for each of the file system's `disks`. All
+# other file system and disk parameters are optional.
+
+# Important:
+# `scale_storage` *must* be define using group variables. Do *not* define disk
+# parameters using host variables or inline variables in your playbook. Doing so
+# would apply them to all hosts in the group/play, thus defining the same disk
+# multiple times...
+
+scale_storage:
+  - filesystem: gpfs01
+    filsets:
```

Comment:
Can you please confirm on the fileset quota size and other value that we were discussing

Reply:
Does filsets: only contain name.  and looks variable name has typo issue

Reply:
Resolved

https://github.com/jayeshh123/ibm-spectrum-scale-install-infra/blob/1504d87ec031d4b1194c3d51b63b530c41841edc/samples/vars/storage_fileset_vars.yml#L19